style.css

```diff
@@ -43,19 +45,189 @@ p {
   margin-bottom: 40px;
 }
 
+ul {
+  font-size: 17px;
+  font-family: "Poppins", sans-serif;
+  margin-left: 40px;
+  font-weight: 500;
+  top: 150px;
+  left: 40px;
+  position: absolute;
+  z-index: 2;
+}
+
+li {
+  color: white;
+}
+
+.main {
+  padding: 104px 16px 16px 16px;
+}
+
 button {
   background-color: rgba(26, 34, 54, 1);
   color: white;
   font-size: 18px;
   font-weight: 500;
   line-height: 24px;
   text-align: center;
-  height: 50px;
-  width: 288px;
+  height: 40px;
+  width: 97%;
   border: none;
+  border-radius: 10px;
 }
 
 .background {
   width: 100%;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.button:hover {
+  color: rgba(26, 34, 54, 1);
+  background-color: white;
+  border: 2px solid rgba(26, 34, 54, 1);
+}
+
+.layout-2 > h1 {
+  font-size: 50px;
+  padding: 40px 0 32px 16px;
+  font-weight: 200;
+  line-height: 50px;
+}
+
+.layout-2 > img {
+  width: 100%;
+  margin: 0;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  background-color: #1a2236;
+  margin: 0;
+}
+
+.wrapper > h2 {
+  color: white;
+  background-color: #1a2236;
+  font-family: "Poppins", sans-serif;
+  font-weight: 300;
+  font-size: 18px;
+  line-height: 70px;
+  margin-top: -5px;
+  padding-left: 16px;
+}
+
+.tech-button {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  column-gap: 0.5em;
+  padding: 20px 60px 0 0;
+  margin-left: 15px;
+  background-color: #1a2236;
+}
+
+.item {
+  background-color: #98dede;
+  border-radius: 3px;
+  border: none;
+  font-size: 10px;
+  line-height: 20px;
+  font-weight: 500;
+  margin-top: -20px;
+}
```

Comment:
- [x] Good job working on the styles but still there is a little issue. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. For the technologies name, there should be a black color for the fonts as on the design but you are using white color Please update the color of the fonts and make sure it matches the design.

Your design

![Screenshot (86)](https://user-images.githubusercontent.com/83803414/153592217-83228745-891b-4f7f-bd85-15177bf224dc.png)


Figma design

![Screenshot (85)](https://user-images.githubusercontent.com/83803414/153592201-c238b4a3-22f6-4202-bc17-f0bb8cd57e81.png)